internal/handlers/handler_test.go

```diff
@@ -37,6 +40,12 @@ import (
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
+type testCase struct {
```

Comment:
That's a common type for query tests, but not for inserts or updates, for example. Let's move it inside TestSize